src/main/java/mate/hwdao/dao/impl/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,197 @@
+package mate.hwdao.dao.impl.jdbc;
+
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.hwdao.dao.CarDao;
+import mate.hwdao.dao.exception.DataProcessingException;
+import mate.hwdao.lib.Dao;
+import mate.hwdao.model.Car;
+import mate.hwdao.model.Driver;
+import mate.hwdao.model.Manufacturer;
+import mate.hwdao.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?)";
+        try (PreparedStatement preparedStatement
+                     = ConnectionUtil.getConnection()
+                .prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setLong(1, car.getManufacturer().getId());
+            preparedStatement.setString(2, car.getModel());
+            preparedStatement.executeUpdate();
+            ResultSet resultSet = preparedStatement.getGeneratedKeys();
+            while (resultSet.next()) {
+                car.setId(resultSet.getObject("GENERATED_KEY", Long.class));
```

Comment:
let's use the same approach in all 3 dao classes

Reply:
done